crates/mun_target/Cargo.toml

```diff
@@ -11,3 +11,6 @@ description = "Describes compilation targets for Mun"
 [dependencies]
 log = "0.4.8"
 failure = "0.1.7"
+
+[dev-dependencies]
+insta="0.16"
```

Comment:
You should really change this setting in CLion to comply with the git standard 😝:
https://stackoverflow.com/questions/16761227/how-to-make-intellij-idea-insert-a-new-line-at-every-end-of-file

Reply:
I work on too many machines 😅